src/ingestion/accounts.rs

```diff
@@ -38,7 +39,13 @@ pub(super) async fn ingest_account(
 ) -> Result<(), IngestionError> {
     let account: account::ActiveModel = account::ActiveModel::try_from(account)?;
 
-    account.insert(db.as_inner()).await?;
-
+    account::Entity::insert(account)
+        .on_conflict(
+            OnConflict::column(account::Column::Id)
+                .update_columns([account::Column::LastModified, account::Column::Balance])
```

Comment:
I think we should be updating a lot more columns here. Actually probably all columns expect the id and createdAt.

Reply:
Resolved in https://github.com/eigerco/quasar/pull/63/commits/8089fbce28c416bd942c26e12903eb644cff7926